package.json

```diff
@@ -165,7 +165,7 @@
     "@typescript-eslint/parser": "^7.3.1",
     "@vscode/l10n-dev": "^0.0.24",
     "@vscode/vsce": "^3.0.0",
-    "esbuild": "^0.15.15",
+    "esbuild": "^0.23.1",
```

Comment:
I had to update esbuild in order to make it bundle browserslist. Browserslist is a dependency by `node_modules/@babel/helper-compilation-targets`.